consensus/babe/babe.go

```diff
@@ -205,10 +195,10 @@ func (b *Session) vrfSign(input []byte) ([]byte, error) {
 }
 
 // BuildBlock Builds the block
-func (b *Session) buildBlock(number *big.Int) (*types.Block, error) {
+func (b *Session) buildBlock(number *big.Int) (types.Block, error) {
 	block := types.Block{
 		Header: types.BlockHeaderWithHash{Number: number},
 		Body:   []byte{1, 2, 3, 4, 5},
 	}
-	return &block, nil
+	return block, nil
```

Comment:
why is this not a ptr anymore?

Reply:
This should be a pointer still. Not sure why I did that. Will fix.